eZ/Publish/Core/Persistence/Legacy/Tests/Content/MapperTest.php

```diff
@@ -676,6 +713,20 @@ protected function getRelationCreateStructFixture()
         return $struct;
     }
 
+    protected function getEventDispatcher(): EventDispatcherInterface
+    {
+        $eventDispatcher = new EventDispatcher();
+        $eventDispatcher->addSubscriber(
+            new ResolveVirtualFieldSubscriber(
+                $this->getValueConverterRegistryMock(),
+                new StorageRegistry([]),
```

Comment:
Why using the class directly instead of mocking it?

Reply:
If the storage isn't used, and we don't need stub functionality (checking method calls), then it's fine to use the class directly. Unless there is an interface declared for `StorageRegistry`, in which case I'd also prefer a mock of that interface.